gen/dcos-config.yaml

```diff
@@ -766,21 +766,6 @@ package:
                   "Port": 8123,
                   "Uri": "/v1/version",
                   "Role": ["master"]
-              },
-              {
-                  "Port": 15055,
-                  "Uri": "/history/last",
-                  "Role": ["master"]
-              },
-              {
-                  "Port": 15055,
-                  "Uri": "/history/minute",
-                  "Role": ["master"]
-              },
-              {
-                  "Port": 15055,
-                  "Uri": "/history/hour",
-                  "Role": ["master"]
```

Comment:
What's the reason for removing these endpoints? This change doesn't seem to relate to the other changes in the PR, and the ticket linked in the description doesn't seem to cover it.

Reply:
oh good point! i will update the description here.
basically this change introduces a test failure if one of the endpoint did not respond with 200 code. history service is not listening on 15055 port [anymore](https://github.com/dcos/dcos/commit/e2f986a49759a1109b17657b8a4753e1008a32c2#diff-ee8c631a232fb776584c0b43f493138e). We did not notice it because 1. support team does not use diagnostics bundles in OSS, 2. We did not check for 200 code response :|